repo/packages/B/beta-cassandra/5/package.json

```diff
@@ -5,8 +5,8 @@
   "minDcosReleaseVersion": "1.9",
   "name": "beta-cassandra",
   "packagingVersion": "4.0",
-  "postInstallNotes": "DC/OS Cassandra is being installed!\n\n\tDocumentation: https://docs.mesosphere.com/latest/usage/service-guides/cassandra\n\tIssues: https://docs.mesosphere.com/support/",
-  "postUninstallNotes": "DC/OS cassandra is being uninstalled.",
+  "postInstallNotes": "DC/OS Cassandra is being installed!\n\n\tDocumentation: https://docs.mesosphere.com/latest/usage/service-guides/cassandra\n\tIssues: https://docs.mesosphere.com/support/\n\n\tIf you installed from the UI, enter `dcos package install beta-cassandra --cli` from the DC/OS CLI to install the Beta Cassandra CLI subcommands.",
+  "postUninstallNotes": "DC/OS Cassandra has been uninstalled.\nFor DC/OS versions from 1.10, no further action is required. For older DC/OS versions, follow the instructions at https://docs.mesosphere.com/1.9/deploying-services/uninstall/#framework-cleaner to remove any persistent state if required.",
```

Comment:
In the `0.30.X` branch we have been using:
```
  "postUninstallNotes": "The DC/OS Apache Cassandra service is being uninstalled.\n\nFor DC/OS versions from 1.10 no further action is required. For older DC/OS versions follow the instructions at {{documentation-path}}uninstall to remove any persistent state if required."
```
Would it make sense to aim for uniformity here?

Reply:
@elezar and I discussed this via DM -- we're OK to leave things as is for now.

Reply:
(one question here: the "has been" implies that the uninstallation is complete, but generally it takes a bit longer for the scheduler to be restarted in uninstall mode so that this can happen)

Reply:
fair point. changing.